changelogs/fragments/0-clickhouse_info.yml

```
@@ -0,0 +1,2 @@
+major_changes:
+- clickhouse_info - removed support for clickhouse versions 21 and 22.
```

```suggestion
major_changes:
- clickhouse_info - removed support for clickhouse versions 21 and 22 (https://github.com/ansible-collections/community.clickhouse/pull/93).

minor_changes:
- clickhouse_info - retrieves and retrieves more fields (https://github.com/ansible-collections/community.clickhouse/pull/93).
```
isn't it what we're doing here? should we specify at least a few `like comments on databases, ..., etc`?